### PR TITLE
fix: add technology icons and update word contents

### PR DESCRIPTION
**Pull request by Fine0830**
Add technology icons and update word contents.

Screenshots

Signed-off-by: Qiuxia Fan <qiuxiafan@apache.org>

### CODE REVIEW

File: src/locales/lang/en.ts [hunk 1]
```diff
@@ -197,7 +199,7 @@ const msg = {
   dayCutTip: "Last 1 day",
   weekCutTip: "Last 1 week",
   monthCutTip: "Last 1 month",
-  serverZone: "Server Zone",
+  serverZone: "Time Zone",
```
Comment (wu-sheng) [inline src/locales/lang/en.ts:202]:
```suggestion
  serverZone: "OAP Server Time Zone",
```
Is this suitable?

File: src/locales/lang/zh.ts [hunk 2]
```diff
@@ -199,7 +201,7 @@ const msg = {
   dayCutTip: "最近1天",
   weekCutTip: "最近1周",
   monthCutTip: "最近1月",
-  serverZone: "服务器时区",
+  serverZone: "时区",
```
Comment (wu-sheng) [inline src/locales/lang/zh.ts:204]:
```suggestion
  serverZone: "OAP Server 时区",
```

File: src/locales/lang/en.ts (hunk 1 above)
Comment (Fine0830) [reply src/locales/lang/en.ts:202]: Yes. It's fine.

Comment (wu-sheng) [reply src/locales/lang/en.ts:202]: Could you take a review locally to see the layout? Whether is it too long?

Comment (Fine0830) [reply src/locales/lang/en.ts]: > Could you take a review locally to see the layout? Whether is it too long?

I have updated the style.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**